hack/make.sh

```diff
@@ -30,9 +30,13 @@ function install_pouch ()
 	
 	# install lxcfs
 	echo "Install lxcfs"
-	apt-get install lxcfs
-	# MUST stop lxcfs service, so pouchd could take over it.
-	service lxcfs stop
+	if grep -qi "ubuntu" /etc/issue ; then
+		apt-get install lxcfs
```

Comment:
In ubuntu:14.04 `apt-get install lxcfs` will fail, as lollows
```
root@xxx:~# apt-get isntall lxcfs 
E: Invalid operation isntall
root@host14:~# apt-get install lxcfs 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package lxcfs
```

os:
Linux host14 4.4.0-93-generic #116~14.04.1-Ubuntu SMP Mon Aug 14 16:07:05 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux